docs/duration.md

```diff
@@ -289,15 +278,8 @@ If `duration` is not a `Temporal.Duration` object, then it will be converted to
 
 If `other` is larger than `duration` and the subtraction would result in a negative duration, the method will throw a `RangeError`.
 
-In order to be valid, the resulting duration must not have fields with mixed signs.
-However, before the result is balanced, it's possible that the intermediate result will have one or more negative fields while the overall duration is positive, or vice versa.
-For example, "4 hours and 15 minutes" minus "2 hours and 30 minutes" results in "2 hours and &minus;15 minutes".
-The `overflow` argument tells what to do in this case:
-
-- In `constrain` mode (the default), subtractions that result in mixed-sign fields will balance those fields with the next-highest field so that all the fields of the result are constrained to have the same sign.
-- In `balance` mode, all fields are balanced with the next highest field, no matter if they have mixed signs or not.
-
-For usage examples and a more complete explanation of how balancing works and why it is necessary, especially for subtracting `Temporal.Duration`, see [Duration balancing](./balancing.md#duration-arithmetic).
+In order to be valid, the resulting duration must not have fields with mixed signs, and so the result is balanced.
```

Comment:
When duration rounding in add/subtract lands, this text should probably be clarified (e.g. to define what "balanced" means, and to explain that balancing will only happen up to the largest unit), but no need to worry about it now.